firefox-ios/Client/Frontend/Browser/Tabs/Views/RemoteTabsViewController.swift

```diff
@@ -489,4 +502,19 @@ class RemoteTabsTableViewController: UITableViewController,
             }
         }
     }
+
+    private func updateInsets() {
+        if shouldUseiPadSetup() {
+            (emptyView as? InsetUpdatable)?.updateInsets(top: view.safeAreaInsets.top, bottom: 0)
```

Comment:
The `RemoteTabsEmptyViewProtocol` could be `InsetUpdatable` to avoid this cast? Both `RemoteTabsEmptyView` and `ExperimentRemoteTabsEmptyView` conforms to `InsetUpdatable`

```suggestion
            emptyView.updateInsets(top: view.safeAreaInsets.top, bottom: 0)
```